for_motion_planner: false
display_loc_edt: true
display_loc_ogm: false
display_glb_edt: false
display_glb_ogm: true
vis_interval: 1
vis_height: 0.5

# profile the rms every time the visualization msg is published
profile_loc_rms: false
profile_glb_rms: false
log_name: "test.csv"

occupancy_threshold: 180

voxel_width: 0.20

local_size_x: 16
local_size_y: 16
local_size_z: 2

ogm:
  min_height: 0.2
  max_height: 2

wave:
  fast_mode: true
  cutoff_dist: 5

# params of hash table
hash:

  bucket_max: 20000

  block_max: 21997
